styles.css

```diff
@@ -0,0 +1,3 @@
+h1{
+    color: aquamarine;
+}
```

Comment:
- [ ] Kindly link your css file to html file so you can see styles displayed on your browser

Reply:
> * [ ]  Kindly link your CSS file to an HTML file so you can see styles displayed on your browser

- [OPTIOANL ] Great job on implementing this change as required, you have linked your CSS file with the HTML but I think you have forgotten to rename the file path, I think you have changed the style.css to stylesheet1.css, fix that as you go to the next project.